bin/assigner/src/main.cpp

```diff
@@ -614,6 +620,7 @@ int main(int argc, char *argv[]) {
             ("private-input,p", boost::program_options::value<std::string>(), "Private input file")
             ("assignment-table,t", boost::program_options::value<std::string>(), "Assignment table output file")
             ("circuit,c", boost::program_options::value<std::string>(), "Circuit output file")
+            ("dump-public-input,d", boost::program_options::value<std::string>(), "Processed public input output file")
```

Comment:
Maybe better to set it as generate-type, not as separate flag?

Reply:
we should provide file name for dump, not only mode